client/core/core.go

```diff
@@ -6282,6 +6321,13 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 // cached, no new requests will be made.
 func (c *Core) cacheRedemptionFeeSuggestion(t *trackedTrade) {
 	// Try to find any book that might have the fee.
+	if rater, is := t.wallets.toWallet.feeRater(); is {
+		if feeRate, err := rater.FeeRate(); err == nil {
+			atomic.StoreUint64(&t.redeemFeeSuggestion, feeRate)
+		} else {
```

Comment:
You want to return after `atomic.StoreUint64(&t.redeemFeeSuggestion` right? (and eliminate the else).  Otherwise it gets the rate from book or server anyway and overwrites this value.